backend/windmill-worker/src/bash_executor.rs

```diff
@@ -292,6 +299,17 @@ Get-Module -ListAvailable | Import-Module
 $env:PSModulePath = \"{}:$PSModulePathBackup\"",
         POWERSHELL_CACHE_DIR
     );
+
+    #[cfg(windows)]
+    let profile = format!(
+        "#$PSModuleAutoloadingPreference = 'None'
+$PSModulePathBackup = $env:PSModulePath
+#$env:PSModulePath = \"C:\\WINDOWS\\System32\\WindowsPowerShell\\v1.0\\Modules;C:\\Program Files\\PowerShell\\7\\Modules\"
```

Comment:
@HugoCasa not quite sure what's going on here, specifically 

```
$env:PSModulePath = ($Env:PSModulePath -split ':')[-1]
Get-Module -ListAvailable | Import-Module	Get-Module -ListAvailable | Import-Module
```
Why do we pick the last element of the list of module paths to import? With powershell 7 in a clean windows install there are a bunch of other paths one could import modules from.

Reply:
best way to do that for both unix and windows would be to use $PSHOME/Modules instead of last path in PSModulePath.
The goal here is to only preload the default modules.
source: https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_psmodulepath?view=powershell-7.4

Reply:
@alpetric  https://github.com/windmill-labs/windmill/pull/4448